python/taichi/lang/ndarray.py

```diff
@@ -84,11 +81,32 @@ def shape(self):
 
     @python_scope
     def __setitem__(self, key, value):
-        return self.arr.__setitem__(key, value)
+        self.arr.__setitem__(key, value)
 
     @python_scope
     def __getitem__(self, key):
         return self.arr.__getitem__(key)
 
     def __repr__(self):
         return '<ti.ndarray>'
+
+
+class NdarrayHostAccess:
+    """Class for accessing VectorNdarray/MatrixNdarray in Python scope.
+    Args:
+        arr (Union[VectorNdarray, MatrixNdarray]): See above.
+        indices_first (Tuple[Int]): Indices of first-level access.
```

Comment:
What are first level and second level?

Reply:
First level refers to coordinates in the field, while second level refers to indices inside the vector/matrix. Maybe I should make it clearer?

Reply:
Using first/second is to tell users this is what they write in the first/second pair of brackets.

Reply:
Explanations are added.